R/config.R

```diff
@@ -160,7 +160,7 @@ get_filter_commit_size <- function(project_name) {
 #' @export
 get_uctags_line_types <- function(project_name) {
 
-  conf_path <- paste0("../conf/", project_name, ".yml")
+  conf_path <- paste0("conf/", project_name, ".yml")
   conf <- yaml::read_yaml(conf_path)
 
   kinds <- conf[["tool"]][["uctags"]][["keep_lines_type"]]
```

Comment:
you should add a return() function at the end of every function so what is returned is explicit.

Reply:
Should I inform the user if the get function returns NULL?

Reply:
Throwing an error that the path could not be found would here, or in the others that the attribute does not exist in the config file would be helpful, yes.